cwiczenie3/osoba.php

```diff
@@ -6,6 +6,7 @@ class Osoba
     private $plec;
     private $rok;
     private $przyjaciel;
+    private $transport = ["pociag", "samochod"];
```

Comment:
To jest błąd - pamiętaj, że `$transport` nie jest tablicą tylko obiektem `Transport` ustawianym na Osobie. To co teraz zrobiłeś mówi o tym, że Osoba ma od razu przypisane dwa transporty (w postaci stringa :(). Powinno być: `private $transport;` albo dodatkowo okraszone adnotacją w postaci komentarza (storm zrozumie wtedy, że na tej własności będziesz ustawiał Transport):
```php
/** @var Transport */ 
private $transport;
```